src/backend/distributed/executor/multi_utility.c

```diff
@@ -290,6 +291,19 @@ multi_ProcessUtility(Node *parsetree,
 		}
 	}
 
+	/* handle DDL on the worker node */
+	if (!schemaNode)
```

Comment:
Ok, I have brought `enable_ddl_propagation = off` calls. We can add the checks when we do #793